README.md

```diff
@@ -744,7 +749,21 @@ This macro returns the sql required to remove duplicate rows from a model or sou
     relation=source('my_source', 'my_table'),
     group_by="user_id, cast(timestamp as day)",
     order_by="timestamp desc",
-    relation_alias="my_cte"
+   )
+}}
+```
+
```

Comment:
Love that there are usage examples for:
* a `source()` relation
* CTE
* `partition_by` containing multiple expressions

To round out the usage examples, it would be awesome to include:
* a `ref()` relation
* `order_by` containing multiple expressions

Shooting from the hip, the following attempts to do both in one example:
```sql
{{ dbt_utils.deduplicate(
    relation=ref('my_model'),
    partition_by="user_id",
    order_by="effective_date desc, effective_sequence desc",
   )
}}
```